include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -1046,10 +1046,8 @@ def AIE_DMAOp: AIE_Op<"dma", [
   }];
 }
 
-// MemOps are not actually Callable, but we want to inline code into them, so we have to
-// implement CallableOpInterface
```

Comment:
Is this some kind of design documentation or precondition that should be kept somewhere, perhaps around the operation description?